code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -1101,19 +1100,17 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 			continue
 		L << "<span class='inathneq'>\"V yraq lbh zl nvq, punzcvba! Yrg tybel thvqr lbhe oybjf!\"</span>\n\
 		<span class='notice'>Inath-Neq's power flows through you!</span>"
-		L.maxHealth += 500
-		L.health += 500
 		L.color = "#1E8CE1"
+		L.fully_heal()
 		L.stun_absorption = TRUE
-		animate(invoker, color = initial(invoker.color), time = 100)
+		L.status_flags |= GODMODE
+		animate(invoker, color = initial(invoker.color), time = 150, easing = EASE_IN)
 		affected_servants += L
-	for(var/i in 1 to 10)
-		sleep(10)
-		for(var/mob/living/L in affected_servants)
-			L.maxHealth -= 50
-			L.health -= 50
-			if(L.maxHealth == initial(L.maxHealth))
-				L.stun_absorption = FALSE
+	sleep(150)
+	for(var/mob/living/L in affected_servants)
+		L << "<span class='notice'>You feel Inath-Neq's power fade from your body.</span>"
+		L.status_flags &= ~GODMODE
+		L.stun_absorption = FALSE
```

Comment:
`addtimer()` isn't always better. It requires making a completely new proc if you're coding from scratch, and sometimes a `spawn()` or `sleep()` is more efficient.